sentry_kube/cli/debug.py

```diff
@@ -0,0 +1,125 @@
+import subprocess
+
+import click
+import json
+import tempfile
+
+from libsentrykube.utils import ensure_kubectl, should_run_with_empty_context
+
+
+__all__ = ("debug",)
+
+
+@click.command(
+    add_help_option=False,
+    context_settings=dict(
+        allow_extra_args=True,
+        allow_interspersed_args=False,
+        ignore_unknown_options=True,
+    ),
+)
+@click.option("--container", "-c")
+@click.option("--image", "-i")
+@click.option("--namespace", "-n", default="default")
+@click.option("--quiet", "-q", is_flag=True)
+@click.pass_context
+def debug(ctx, container, image, namespace, quiet):
+    """
+    Convenient wrapper for running "kubectl debug" with env and volume mounts
+    """
+    args = ctx.args
+    context = ctx.obj.context_name
+    customer = ctx.obj.customer_name
+    quiet = ctx.obj.quiet_mode or quiet
+
+    cmd = [f"{ensure_kubectl()}"]
+    if not should_run_with_empty_context():
+        cmd += ["--context", context]
+
+    pod_name = args[0]
+
+    from kubernetes import client as k8s_client
+    from libsentrykube.kube import kube_get_client
+
+    api = k8s_client.CoreV1Api(kube_get_client())
+    api_client = k8s_client.ApiClient()
+
+    # read spec of the target pod
+    resp = api.read_namespaced_pod(namespace=namespace, name=pod_name)
+    # convert to camelCase
+    resp = api_client.sanitize_for_serialization(resp)
+
+    # for writing the custom debug profile
+    tmp = tempfile.NamedTemporaryFile(delete=True)
+
+    if container:
+        container_spec = next(
+            (item for item in resp["spec"]["containers"] if item["name"] == container),
+            None,
+        )
+    else:
+        container_spec = resp["spec"]["containers"][0]
```

Comment:
Do we want to try to be a little more clever here for defaults and explicitly exclude pgbouncer/envoy containers that we typically wouldn't want to connect to by default?